[2022-09-14T09:37:31.187+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: vecap_etl.date_dim scheduled__2022-09-13T00:00:00+00:00 [queued]>
[2022-09-14T09:37:31.208+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: vecap_etl.date_dim scheduled__2022-09-13T00:00:00+00:00 [queued]>
[2022-09-14T09:37:31.210+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-14T09:37:31.211+0000] {taskinstance.py:1369} INFO - Starting attempt 2 of 4
[2022-09-14T09:37:31.212+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-14T09:37:31.255+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): date_dim> on 2022-09-13 00:00:00+00:00
[2022-09-14T09:37:31.270+0000] {standard_task_runner.py:52} INFO - Started process 260 to run task
[2022-09-14T09:37:31.285+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vecap_etl', 'date_dim', 'scheduled__2022-09-13T00:00:00+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/c_dag.py', '--cfg-path', '/tmp/tmpkcxzb5r7', '--error-file', '/tmp/tmpsyw2k242']
[2022-09-14T09:37:31.290+0000] {standard_task_runner.py:80} INFO - Job 81: Subtask date_dim
[2022-09-14T09:37:31.479+0000] {task_command.py:371} INFO - Running <TaskInstance: vecap_etl.date_dim scheduled__2022-09-13T00:00:00+00:00 [running]> on host 0d275337f52b
[2022-09-14T09:37:31.699+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@vecap.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vecap_etl
AIRFLOW_CTX_TASK_ID=date_dim
AIRFLOW_CTX_EXECUTION_DATE=2022-09-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-13T00:00:00+00:00
[2022-09-14T09:37:39.915+0000] {python.py:173} INFO - Done. Returned value was: <sqlalchemy.engine.cursor.LegacyCursorResult object at 0x7f73da2e5410>
[2022-09-14T09:37:39.954+0000] {xcom.py:586} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.
[2022-09-14T09:37:39.958+0000] {taskinstance.py:1902} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2405, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 68, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 198, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 583, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type LegacyCursorResult is not JSON serializable
[2022-09-14T09:37:39.996+0000] {taskinstance.py:1412} INFO - Marking task as UP_FOR_RETRY. dag_id=vecap_etl, task_id=date_dim, execution_date=20220913T000000, start_date=20220914T093731, end_date=20220914T093739
[2022-09-14T09:37:40.060+0000] {standard_task_runner.py:97} ERROR - Failed to execute job 81 for task date_dim (Object of type LegacyCursorResult is not JSON serializable; 260)
[2022-09-14T09:37:40.130+0000] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-09-14T09:37:40.242+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
